### PR TITLE
Replace RestClient with Excon

### PR DESCRIPTION
**Pull request by ruianderson**

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -2,7 +2,6 @@
 require 'json'
 require 'active_support/core_ext/array/conversions'
 require 'active_support/core_ext/hash/conversions'
-require 'rest_client'
```
Comment (dwayne) [inline app/app.rb]: @ruianderson Are you missing a `require 'excon'`?

Comment (ruianderson) [reply app/app.rb]: Good catch. I'm wondering how the tests passed :thinking:

File: lib/requester.rb [hunk 2]
```diff
@@ -12,8 +12,10 @@ class << self
     # @param content_type [String]
     # @return [Hash] Patron response
     def request(endpoint, body, content_type)
-      # RestClient.proxy = 'http://199.200.120.37:7808'
-      RestClient.post endpoint, body, content_type: content_type
+      Excon.post(endpoint, {
```
Comment (dwayne) [inline lib/requester.rb]: @ruianderson Maybe the tests don't actually run this method and instead mock it out.

Comment (ruianderson) [reply lib/requester.rb]: No, they run since the VCR were rerecorded

Comment (dwayne) [reply lib/requester.rb]: Maybe another gem uses excon. Any third party library that loads excon makes it available to your code as well.

Comment (dwayne) [reply lib/requester.rb]: Here: https://rubygems.org/gems/webmock. Webmock uses excon.

Comment (dwayne) [reply lib/requester.rb]: (a) https://github.com/bblimke/webmock/blob/146c5c2d8a05d5277843841beb8e56d1b998bd19/lib/webmock.rb#L55
(b) https://github.com/bblimke/webmock/blob/d8686502442d9830dcccd24a1120ac08413d857a/lib/webmock/http_lib_adapters/excon_adapter.rb#L2

Spooky action at a distance. One of the downsides of Ruby.

Comment (ruianderson) [reply lib/requester.rb]: :+1: that's true. I thought about that too. But ok, that's fine to have the require explicit

File: lib/requester.rb [hunk 3]
```diff
@@ -12,8 +12,10 @@ class << self
     # @param content_type [String]
     # @return [Hash] Patron response
```
Comment (dwayne) [inline lib/requester.rb]: Not high priority but I noticed that the comment is outdated as it mentions Patron.

**Comment by dwayne** [comment]
I approve it. LGTM. \cc @jstotz

**Comment by jstotz** [comment]
@ruianderson You can go ahead and merge this once you're ready to test it.

**Pull request merged**